lib/teleterm/gateway/kube.go

```diff
@@ -209,3 +210,15 @@ func (k *kube) writeKubeconfig(key *keys.PrivateKey, cas map[string]tls.Certific
 	})
 	return nil
 }
+
+func (k *kube) CLICommand() (*api.GatewayCLICommand, error) {
+	// TODO(greedy52) currently kube must implement CLICommand in order to pass
+	// Kube to CLICommandProvider. We should revisit gateway design/flows like
```

Comment:
Could you add a test which captures this behavior? If I remove this method, the code compiles and both unit and integration tests pass.